src/explore-education-statistics-admin/src/pages/release/pre-release/components/PreReleaseInvitePlanModal.tsx

```diff
@@ -0,0 +1,120 @@
+import styles from '@admin/pages/release/pre-release/components/PreReleaseInvitePlanModal.module.scss';
+import ModalConfirm from '@common/components/ModalConfirm';
+import WarningMessage from '@common/components/WarningMessage';
+import React from 'react';
+import { PreReleaseInvitePlan } from '@admin/services/preReleaseUserService';
+
+interface Props {
+  invitePlan: PreReleaseInvitePlan;
+  isReleaseApproved: boolean;
+  onConfirm: () => void;
+  onCancel: () => void;
+  onExit: () => void;
+}
+
+const PreReleaseInvitePlanModel = ({
+  invitePlan,
+  isReleaseApproved,
+  onConfirm,
+  onCancel,
+  onExit,
+}: Props) => {
+  if (!invitePlan) {
```

Comment:
Is this actually optional? The prop type isn't optional, so it doesn't seem like we need this check?